<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc v2.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: ubuntu;
	font-size: 30px;
	font-weight: 0px;
	color: white;
	text-align: center;
	vertical-align: center;
	border-radius: 22px;
	border: 0px;
}

QWidget {
	background-color: #17181a;
}

QPushButton {
	background-color: #1991ff;
	color: #b2daff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}

QSpinBox {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}

QDoubleSpinBox {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}

QDateEdit {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}

QDateEdit::up-arrow {
	border-radius: 15px;
	border-image: url(:/icons/icons/arrow_up.png);
	padding-right: 10px;
}

QDateEdit::down-arrow {
	border-radius: 15px;
	border-image: url(:/icons/icons/arrow_down.png);
	padding-right: 10px;
}

QDateEdit {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}

QComboBox {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
	padding-left: 10px;
	padding-right: 10px;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-size: 25px;
	border-radius: 28px;
	background-color: #303136;
	color: #24a5ff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}

QPushButton:disabled {
	background-color: #868686;
	color: #5a5a5a;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/calc_off.png</normaloff>:/icons/icons/calc_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="graphicButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/graphic_off.png</normaloff>:/icons/icons/graphic_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="creditButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/credit_on.png</normaloff>:/icons/icons/credit_on.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="depositButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/deposit_off.png</normaloff>:/icons/icons/deposit_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>depositButton</zorder>
   <zorder>calcButton</zorder>
   <zorder>creditButton</zorder>
   <zorder>graphicButton</zorder>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Timeframe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Total payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="overpayment">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>580</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_payment">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_payments">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>651</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="calculate_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="timeframe">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QComboBox" name="type">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>annuity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>differential</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Montly payment</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amount">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="minimumDate">
    <date>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>d M yyyy</string>
   </property>
  </widget>
  <widget class="QComboBox" name="frequency">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>monthly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>quartaly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yearly</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	color: red;
	font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
